File: 1sem/Python/Lista2/Satoshi/Ex05.py
def main():
    v:int = [0]*20
    soma:int = 0
    i:int

    for i in range(20):
        v[i] = int(input(f'Digite o valor na posição {i+1}: '))
        soma += v[i] - v[19-i]
    print(soma)

if __name__ == '__main__':
    main()

File: 2sem/Java/FuncoesRecursivas/prj_java16-09-2026/src/Ex05.java
import javax.swing.JOptionPane;
public class Ex05 {
    static int f;
    public static void main (String args[]){
        int n,t=0;
        
        do {
            n = Integer.parseInt(JOptionPane.showInputDialog("Digite um valor para N: "));
        }while (n < 1);
        
        
        System.out.println("Resultado: "+Soma(n,t));
    }
    
    static int Soma (int n,int t){
        if (n > 1){
            t = Fat(n);
            t = t + Soma(n-1,t);
        } else {
            return 1;
        }
        return t;
    }
    
    static int Fat (int n){
        if (n > 1){
            f = n * Fat(n-1);
        } else{
            return 1;
        }
        return f;
    }
}
